src/vs/workbench/contrib/debug/browser/debugService.ts

```diff
@@ -432,7 +433,23 @@ export class DebugService implements IDebugService {
 							nls.localize('debugTypeMissing', "Missing property 'type' for the chosen launch configuration.");
 					}
 
-					await this.showError(message);
+					let actionList: IAction[] = [];
+					let action = {
+						id: 'Install Debug Extension',
+						label: 'Install Debug Extension(s)',
+						tooltip: 'Install Debug Extension',
+						class: 'string',
+						checked: true,
+						run: async () => {
+							const viewlet = (await this.viewletService.openViewlet(EXTENSIONS_VIEWLET_ID, true))?.getViewPaneContainer() as IExtensionsViewPaneContainer;
```

Comment:
Maybe instead of duplicating code just use the 
`this.commandService.executeCommand('debug.installAdditionalDebuggers')`
For that you will have to introduce the `commandService` dependency to the `debugService` which is fine.

Reply:
This was really helpful! I spent some time and I believe I got this implemented, let me know what you think.